src/index.scss

```diff
@@ -33,7 +33,12 @@ html {
     #root {
       height: 100%;
     }
-
+    .height100 {
```

Comment:
This is great!  I believe there are bootstrap utility classes you can learn to accomplish these, take a look at [this](https://getbootstrap.com/docs/4.0/utilities/sizing/) to get you started 😁

Reply:
Awesome!

Reply:
did this. Thank you!